README.md

```diff
@@ -308,6 +308,32 @@ Using a very high number of connections will introduce overhead that can slow th
 
 Please note that when a file is being saved, it goes through two processes. First, the downloaded file "chunks" are pieced back together to reconstruct the original file. Second, pyEGA3 calculates the checksum of the file to confirm the file downloaded successfully. Larger files will take more time to reconstruct and validate the checksum.
 
+### The --saveto argument was renamed to --output-dir
+
+The `--saveto` command-line argument is now called `--output-dir` and, in contrast with the original
+`--saveto` argument, one can only specify now an output directory, but not an output file.
+
+Thus, the original command, where the `--saveto` argument specified a directory:
+
+```bash
+pyega3 -cf </Path/To/CREDENTIALS_FILE> fetch EGAF<NUM> --saveto </Path/To/OutputDirectory>
+```
+
+should be rewritten like this:
+
+```bash
+pyega3 -cf </Path/To/CREDENTIALS_FILE> fetch EGAF<NUM> --output-dir </Path/To/OutputDirectory>
+```
+
+The original command:
+
+```bash
+pyega3 -cf </Path/To/CREDENTIALS_FILE> fetch EGAF<NUM> --saveto </Path/To/NewFileName>
+```
+
+where the `--saveto` argument specified a new file-name, cannot be rewritten,
+because the `--output-dir` argument supports only output directories, but not output files.
+
```

Comment:
For a rendered version, please click here: https://github.com/EGA-archive/ega-download-client/tree/EE-2210-update-faq#the---saveto-argument-was-renamed-to---output-dir